### DISCUSSION TITLE
希望加一个和MCSM面板一样的简单配置面板

### DISCUSSION THREAD

**Discussion by yangyan-ot** [Meloong-Git/PCL#2633] [提问] [answered]
emm....MCSM(开服务器的面板)有个功能就是可以翻译出服务器配置文件的内容可以任意更改
希望PCL2也可以做一个翻译出mc配置文件并可以任意更改

**Comment by so1ve** [comment, marked as answer]
🤔 请去issue中提交新功能提案谢谢喵

**Comment by Yue-cn** [comment]
若 0基础，他应该不会去改; 他会改，就不会用到这个了。

**Comment by iPodtouch6557** [comment]
个人认为这个问题已经解决了啊，挂在这干嘛
